receiver/hostmetricsreceiver/metadata.yaml

```diff
@@ -293,6 +293,22 @@ metrics:
       monotonic: false
     labels: [network.protocol, network.state]
 
+  system.network.conntrack.count:
```

Comment:
I am not knowledgeable enough about macOS or Windows to know if there are equivalents to conntrack in these OSs. It might be a good idea to check this and make the name more general in case we want to add support for other platforms later.

A good way to get feedback on this is to open a PR on the https://github.com/open-telemetry/opentelemetry-specification repository so that other people can chime in.